sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1633,8 +1647,20 @@ void CAFMaker::produce(art::Event& evt) noexcept {
     //#######################################################
     FillSliceVars(*slice, primary, producer, recslc);
     FillSliceMetadata(primary_meta, recslc);
-    FillSliceFlashMatch(fmatch, recslc);
-    FillSliceFlashMatchA(fmatch, recslc);
+    FillSliceFlashMatch(fmatch_map["fmatch"], recslc.fmatch);
+    FillSliceFlashMatch(fmatch_map["fmatch"], recslc.fmatch_a);
```

Comment:
There's no point in having this copy `recslc.fmatch_a`, nor any of the other copies below.